docs/src/docs/usage/install/index.mdx

```diff
@@ -151,6 +151,32 @@ As such, we recommend using the fixed minor version and fixed or the latest patc
 For example, in our [GitHub Action](https://github.com/golangci/golangci-lint-action) we require users to explicitly set the minor version of `golangci-lint`
 and we always use the latest patch version.
 
+## Linter Deprecation Cycle
+
+Sometimes a linter can be deprecated because:
+
+- The linter stops working with a newer version of Go.
+- The author has abandoned its linter.
```

Comment:
This looks like an exhausive list but there can be many more reasons. I wouldn't make it a list for that reason.

```suggestion
A linter can be deprecated for various reasons, e.g. the linter stops working with a newer version of Go or the author has abandoned its linter.
```